source/versions/filenames.rst

```diff
@@ -0,0 +1,126 @@
+=============================
+Reading the Download Filename
+=============================
+
+.. note::
+
+    Our versioning policy was updated in October 2018. **Beginning with** SpongeAPI 7.1, the following information 
+    applies. See :doc:`legacy-versions` for information on older versions.
+
+When you download files, the name of the file will provide some important version information. The following sections 
+describe the information provided.
+
+.. _sponge-forge-file-name:
+
+SpongeForge
+===========
+
+The format of the SpongeForge download filename is:
+
+``spongeforge-<MCVersion>-<ForgeBuild>-<APIMajor>.<LatestAPIMinorRelease>.<RecommendedVersion(-RC<BuildNumber>)>.jar``
+
++----------------------+-----------------------------------------------------------------------------------------------+
```

Comment:
I'm not sure, but the tables without headers look a little bit strange. Should we add the headers?